README.md

```diff
@@ -115,38 +120,39 @@ You can see collection of all configuration files at *$TIZENRT_BASEDIR/build/con
 To check all pre-defined configurations, type as follows:
 
 ```bash
-./configure.sh --help
+./tools/configure.sh --help
 ```
 
-##### 1.1 Additional Configuration (optional)
-
-After basic configuration by [1. Configuration](#1-configuration), you can additionally modify your configuration with *menuconfig*.
+Additionally, you can modify your configuration with *menuconfig*.
```

Comment:
I think previous is better.
1. This is not mandatory.
2. This should be done after number 1.

Reply:
Agreed. I have addressed the changes.